src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.objects.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        // SHOULD I MAKE RANDOM AS CLASS FIELD OR JUST AS SIMPLE LOCAL FUNCTION VARIABLE ?
+        this.random = new Random();
+        colorSupplier = new ColorSupplier();
+    }
```

Comment:
we should make this fields as class fields because there's no purpose in creating them each time when getRandomBall() method is being invoked. we need to create them just once 
```suggestion
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();

```